code/game/machinery/Sleeper.dm

```diff
@@ -43,13 +44,14 @@
 	for(var/obj/item/weapon/stock_parts/matter_bin/B in component_parts)
 		rating += B.rating - 1
 
+	var/scanning = 0
```

Comment:
This var is already in line 9

Reply:
I was having trouble defining the scanning var for some reason, I added it after testing it on a private server because the one from line 9 for some reason did not count? Thanks for the heads up, I'll clean it up.